vault/osfamilymap.yaml

```
@@ -3,6 +3,9 @@
 ---
 Debian:
   setcap_pkg: libcap2-bin
+  repo: "deb [arch={{ grains['osarch'] }}] https://apt.releases.hashicorp.com {{ grains['oscodename'] }} main"
```

Please split long lines.
```suggestion
  repo: 'deb [arch={{ grains['osarch'] }}] https://apt.releases.hashicorp.com
         {{ grains['oscodename'] }} main'
```

---

fixed. Along other stuffs, to get CI ok.